odoo/custom/src/private/thaiev_report/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "ThaiEV: Report",
+    "summary": "Customize Report template for ThaiEV",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "category": "Thai EV",
+    "author": "Ecosoft",
+    "installable": True,
+    "depends": ["thaiev_install"],
```

Comment:
Why depends on thaiev_install module, you should depend on related modules with this module.

Reply:
Done!